app/scripts/components/mapcomponent.js

```diff
@@ -105,7 +121,7 @@ var exports = (function () {
           },
           "paint": {
             "circle-opacity": 0.5,
-            "circle-color": "#009AEB",
```

Comment:
Please use the old color, or better, try to determine it from the scss ($col-bw-0)